src/main/java/blackjack/domain/participant/Participant.java

```diff
@@ -33,20 +34,82 @@ public Hand getHand() {
 
     // TODO: 구현체의 canReceive 메소드와 상당히 겹침. 해결 필요.
     public boolean isBusted() {
-        return hand.getScore().isGreaterThan(Score.BLACKJACK);
+        return Score.calculateSumFrom(hand).isBusted();
     }
 
-    // TODO: 조건식 단순화
-    public ResultType compareWith(Participant other) {
-        int playerScore = getCurrentScore().getValue();
-        int otherScore = other.getCurrentScore().getValue();
+    public boolean isBlackjack() {
+        List<Card> initialCards = hand.getInitialCards();
+        Score initialScore = Score.calculateSumFrom(initialCards);
 
-        if ((playerScore > otherScore && !isBusted()) || (!isBusted() && other.isBusted())) {
+        return initialScore.equals(Score.valueOf(Score.MAXIMUM_SCORE));
+    }
+
+    protected ResultType compareWith(Participant other) {
```

Comment:
```suggestion
    protected ResultType compareWith(Dealer other) {
```
other에 Player가 들어가도 무관한가요?

Reply:
딜러가 플레이어와 비교할 경우가 있을 것이라고 생각해서 이렇게 구현했는데, 생각해보니 Dealer 로 타입을 제한해도 될 것 같아요. 수정했습니다 :)